controls/roles/manage-service/molecule/ssv-lighthouse/converge.yml

```diff
@@ -8,23 +8,6 @@
     ssv_service: 1ce654a0-7162-11ec-91f3-8fdbc286367e
 
   tasks:
-    # ssv operator keys
-    - block:
-      - name: Generate SSV network keys
-        command: bash -c "docker run -d --name=ssv_node_op_key -it 'bloxstaking/ssv-node:alpine-dns' /go/bin/ssvnode generate-operator-keys &&
-                          docker logs ssv_node_op_key --follow &&
-                          docker stop ssv_node_op_key &&
-                          docker rm ssv_node_op_key"
-        register: ssv_keys
-        changed_when: false
-        become: yes
-      - name: Parse keys
-        set_fact:
-          ssv:
-            enabled: true
-            pk: "{{ ssv_keys.stdout | regex_search('(?<=\"pk\":\\s\").*(?=\")') }}"
-            sk: "{{ ssv_keys.stdout | regex_search('(?<=\"sk\":\\s\").*(?=\")') }}"
-
     # ssv service
     - block:
```

Comment:
shouldn't be the ssv-service after generating the keys and writing the config file? otherwise it wouldn't test it right?

Reply:
that's because of scenario `manage-service` -> `ssv-*`. `Manage-service` creates `<id>.yaml` file. After that we can write `ssv-keys` into `<id>.yaml` file. It works well, log looks fine. there is no need to restart `blox-ssv service`.